aspnetcore/release-notes/aspnetcore-3.0.md

```diff
@@ -0,0 +1,501 @@
+---
+title: What's new in ASP.NET Core 3.0
+author: rick-anderson
+description: Learn about the new features in ASP.NET Core 3.0.
+ms.author: riande
+ms.custom: mvc
+ms.date: 09/19/2019
+uid: aspnetcore-3.0
+---
+# What's new in ASP.NET Core 3.0
+
+This article highlights the most significant changes in ASP.NET Core 3.0 with links to relevant documentation.
+
+## Blazor
+
+Blazor is a new framework in ASP.NET Core for building interactive client-side web UI with .NET:
+
+* Create rich interactive UIs using C# instead of JavaScript.
+* Share server-side and client-side app logic written in .NET.
+* Render the UI as HTML and CSS for wide browser support, including mobile browsers.
+
+Using .NET for client-side web development offers the following advantages:
+
+* Write code in C# instead of JavaScript.
+* Leverage the existing .NET ecosystem of .NET libraries.
+* Share app logic across server and client.
+* Benefit from .NET's performance, reliability, and security.
+* Stay productive with Visual Studio on Windows, Linux, and macOS.
+* Build on a common set of languages, frameworks, and tools that are stable, feature-rich, and easy to use.
+
+Blazor framework supported scenarios:
+
+* Reusable UI components (Razor components)
+* Client-side routing
+* Component layouts
+* Support for dependency injection
+* Forms and validation
+* Build component libraries with Razor class libraries
+* JavaScript interop
+
+For more information, see <xref:blazor/index>.
+
+### Blazor Server
+
+Blazor decouples component rendering logic from how UI updates are applied. Blazor Server provides support for hosting Razor components on the server in an ASP.NET Core app. UI updates are handled over a SignalR connection. Blazor Server is supported in the ASP.NET Core 3.0 release.
+
+### Blazor WebAssembly
```

Comment:
Can we remove this section heading and just have the following paragraph be part of the previous section? I worry that this section heading will give the impression scanning the headings that Blazor WebAssembly is part of .NET Core 3.0 , when it isn't.